services/gateway/app.js

```diff
@@ -139,7 +139,26 @@ broker.createService({
 		'update.fee_estimates': (payload) => sendSocketIoEvent('update.fee_estimates', mapper(payload, feesDefinition)),
 		'coreService.Ready': (payload) => updateSvcStatus(payload),
 	},
-});
+};
+
+if (config.rateLimit.connectionLimit > 0) {
+	logger.info(`Enabling rate limiter, connLimit: ${config.rateLimit.connectionLimit}, window: ${config.rateLimit.window}`);
+	
+	gatewayConfig.settings.rateLimit = {
+		window: config.rateLimit.window || 15 * 1000,
+		limit: config.rateLimit.connectionLimit || 20,
+		headers: true,
+	
+		key: (req) => {
+			return req.headers["x-forwarded-for"] ||
+				req.connection.remoteAddress ||
+				req.socket.remoteAddress ||
+				req.connection.socket.remoteAddress;
```

Comment:
Please fix the linting errors.
```suggestion
			return req.headers["x-forwarded-for"]
				|| req.connection.remoteAddress
				|| req.socket.remoteAddress
				|| req.connection.socket.remoteAddress;
```